client/components/gallery-shortcode/index.jsx

```diff
@@ -83,7 +83,7 @@ export default class GalleryShortcode extends React.Component {
 	};
 
 	getAttributes = () => {
-		let attributes = pick( this.props, 'items', 'type', 'columns', 'orderBy', 'link', 'size' );
+		const attributes = pick( this.props, 'items', 'type', 'columns', 'orderBy', 'link', 'size' );
```

Comment:
There's one possible improvement in the `GalleryShortcode` component that would be an awesome drive-by patch: rewrite this line:
```js
omit( this.props, Object.keys( this.constructor.propTypes ) )
```
to something that doesn't access the component's `propTypes` at runtime. If we can eliminate all code like this, we could remove all proptypes definitions from production bundles.